fabtests/common/shared.c

```diff
@@ -2327,6 +2331,26 @@ ssize_t ft_post_rma(enum ft_rma_opcodes op, char *buf, size_t size,
 			remote_fi_addr, remote->addr + ft_remote_read_offset(buf),
 			remote->key, context);
 		break;
+	case FT_RMA_WRITEMSG:
```

Comment:
I am concerned that, if we want to make `FT_RMA_WRITEMSG` as a separate rma op code, you would expect user to run `fi_rma_bw -o writemsg`, but your change doesn't seem to work for this case. A simpler change could be keep the current op codes, but implement an internal `ft_post_rma_writemsg` that is used for fi_more usage.

Reply:
I can add writemsg to ft_parse_api_opts if you prefer that.

Reply:
The problem is whether we want to test the api explicitly.... similar to inject, we don't make it as an op code, but we use it implicitly

Reply:
That's a valid concern. I think the proposed `ft_post_rma_writemsg` is more preferrable than adding the new op code.